### PR TITLE
add attachment export service

### PR DESCRIPTION
**Pull request by pavel-zhuravlyov**

### CODE REVIEW

File: app/services/exporter.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    @metadata.present?
+  end
+
+  def archive_data
+    time_range =
+      if start_time.present? && end_time.present?
+        "#{start_time}_#{end_time}"
+      else
+        'all'
```
Comment (sudodoki) [inline app/services/exporter.rb]: might add current date as end so you could know when the export was ran

Comment (pavel-zhuravlyov) [reply app/services/exporter.rb]: all archives have a timestamp (e.g. hrms_data_all_20191205175239.zip in this case)

File: app/services/exporter.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  def perform(start_time: nil, end_time: nil)
+    collect_data(start_time, end_time) && archive_data
+  end
+
+  def collect_data(start_time, end_time)
+    @metadata_fields_types = {
+      person: PERSON_FIELDS_TO_EXPORT.map { |attr| { name: attr, type: Person.columns_hash[attr.to_s].type } }
+    }
+
+    attachments = Attachment.where('name ILIKE ?', '%cv%')
```
Comment (sudodoki) [inline app/services/exporter.rb]: @rilian are we 100% sure those are all the CVs? :thinking: maybe, some dropdown with type of attachemt (or checkbox is_cv) should be used instead. Wonder whether we have any cases where cyrillic c is mixed with v

File: app/services/exporter.rb (hunk 1 above)
Comment (sudodoki) [reply app/services/exporter.rb]: oh, my bad, this is awesome

File: app/services/exporter.rb (hunk 2 above)
Comment (rilian) [reply app/services/exporter.rb]: we dont use cyrillic and most probably yes, if name contains cv it is a cv

File: spec/services/exporter_spec.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+describe Exporter do
+  subject { described_class.new }
+
+  EXPECTED_FILE_METADATA_KEYS = %i[id name content_type metadata].freeze
```
Comment (rilian) [inline spec/services/exporter_spec.rb]: why freeze if you already added a comment on line 1

**Comment by sudodoki** [comment]
seems ok from my side.

File: spec/services/exporter_spec.rb (hunk 3 above)
Comment (pavel-zhuravlyov) [reply spec/services/exporter_spec.rb]: rubocop asked to freeze it

**Comment by pavel-zhuravlyov** [comment]
fine with me

**Pull request merged**